admin/preprint_providers/views.py

```diff
@@ -23,8 +23,6 @@
 from osf.models.provider import rules_to_subjects, WhitelistedSHAREPreprintProvider
 from website import settings as osf_settings
 
-# When preprint_providers exclusively use Subject relations for creation, set this to False
-SHOW_TAXONOMIES_IN_PREPRINT_PROVIDER_CREATE = True
```

Comment:
A) Why not just set this to false? 
B) Are we now exclusively using subject relations for creation? 
C) Why is this new with the changes related to this PR (and the previous one that this is fixing) when it was working before?

Reply:
A) Persumably it's vestigial, I think it's just a pre-waffle thing.

B) I don't understand exactly what you mean by this, but creating a preprint provider and adding a custom taxonomy are two different steps. Before it was possible to do both these from the same page on preprint provider creation, but I have to assume it would just break anyway, I doubt anyone tried it.

C) `create.html` and `update_preprint_provider_form.html` are being merged (because admin templates are now composed differently). This makes loading the link for the custom taxonomy page is impossible before the preprint provider exists (because no pk for link). So when just don't load the custom taxonmy link when creating a preprint provider, but do when editing.